uwsgi.ini

```diff
@@ -11,5 +12,6 @@ enable-threads = true
 single-interpreter = true
 if-env = HTTP_AUTH_CREDENTIALS
 plugins = python,router_basicauth
+route = ^/api/v\d+/order_fulfillment/? continue:
```

Comment:
I'm not sure the `\d+` is necessary, we don't have any `v1` URLs at the moment and when we do they may have a different structure from the `v0` ones